7주차/24499/24499_python_토르.py

```diff
@@ -0,0 +1,35 @@
+from collections import deque
+import sys
+import copy
+from collections import defaultdict
+
+input = sys.stdin.readline
+N, K = map(int, input().split())
+pies = list(map(int, input().split()))
+answer = 0
+
+cur_pies = []
+tmp_sum = 0
+left, right = 0, 0
+
+while len(cur_pies) < K:
+    cur_pies.append(pies[right])
+    tmp_sum += pies[right]
+    right += 1
+    if right >= len(pies):
+        right %= len(pies)
```

Comment:
`cur_pies`가 여기서만 사용되네요. 해당 로직의 용도는 `tmp_sum`을 구하기 위함인가요???
그렇다면 `tmp_sum = sum(pies[:K])`으로 대체하면 어떨까요?

Reply:
아하, right의 초기값 세팅이 여기서 이뤄지는군요!!
그렇다면!! 아래와 같이 바꿀 수 있겠네요
```suggestion
right = K
tmp_sum = sum(pies[:K])
```